x-pack/plugins/cases/public/components/user_action_tree/index.tsx

```diff
@@ -154,9 +161,9 @@ export const UserActionTree = React.memo(
     const handleManageMarkdownEditId = useCallback(
       (id: string) => {
         if (!manageMarkdownEditIds.includes(id)) {
-          setManangeMardownEditIds([...manageMarkdownEditIds, id]);
+          setManageMarkdownEditIds([...manageMarkdownEditIds, id]);
```

Comment:
To avoid weird behavior with depedency
```suggestion
          setManageMarkdownEditIds((prevMmanageMarkdownEditIds) => ([...prevMmanageMarkdownEditIds, id]));
```